### PR TITLE
PPTT-1989 Added AMP integration page

### PR DESCRIPTION
**Pull request by RomanSek**
Preview:
https://developers.piwik.pro/en/feature-pptt-1989-amp-integration/integrations/AMP_integration.html

Old SDK page now redirects to integrations:
https://developers.piwik.pro/en/feature-pptt-1989-amp-integration/sdk/

### CODE REVIEW

**Review by JacekClearcode — APPROVED** [review]
So we are changing SDK to integrations... huh?

**Comment by RomanSek** [comment]
> So we are changing SDK to integrations... huh?

Yes, we're reorganizing that section to something more general.

File: integrations/AMP_integration.rst [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+
+Basic setup
+-----------
+
+This setup allows you to track page views. Copy following code to your AMP page while replacing:
+
+* ``<INSTANCE_DOMAIN>`` - PPAS instance domain (e.g. ``analytics.example.com``)
+* ``<APP_ID>`` - PPAS application ID (e.g. ``12345678-1234-1234-1234-1234567890ab``)
+* ``<TRACKER_HASH>`` - If there is no non-AMP page tracked by traditional tracker this value may be removed from configuration. To guarantee that same cookie will be used by AMP and non-AMP pages on page domain, this value should be taken from the name of the ID cookie generated by JS tracker. Check :ref:`how to get cookie hash<howToGetTrackerCookieHash>` section for detailed instructions.
```
[the remark this answers is not in the document]
Comment (quba) [reply integrations/AMP_integration.rst:13]: is there maybe a better method to get the proper hash for given domain? Not sure where this hash() function comes from, but that would make it easier for the dev to implement it without installing the code on prod.

File: integrations/AMP_integration.rst [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+------------------------------
+Each JS tracker generates unique cookie name based on its configuration. Follow these instructions to get hash from cookie generated by JS tracker:
+
+* Setup JS tracker on non-AMP page (if it was not done already).
+* Open tracked page in the browser.
+* Open console and look for cookie starting with ``_pk_id.``. Cookie name should look like this: ``_pk_id.12345678-1234-1234-1234-1234567890ab.1234``. The part after first dot is the value of App ID of the cookie (if there are multiple cookies starting with ``_pk_id.`` it may be used to identify correct cookie). Next part after second dot is the cookie hash generated by tracker. Copy this part and replace ``<TRACKER_HASH>`` with it.
+
+Here you can see how to look for tracker cookie in Google Chrome console:
+
+.. image:: /_static/images/amp_integration/cookie-hash.png
```
[the remark this answers is not in the document]
Comment (RomanSek) [reply integrations/AMP_integration.rst:42]: Changed

File: integrations/AMP_integration.rst [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+
+* “`selector <https://www.ampproject.org/docs/reference/components/amp-analytics#element-selector>`_” - CSS selector for element that should be watched
+* “on” - HTML event type
+* “vars” - Variables that should be used by this event. Custom events expect:
+* “event_category” - required
+* “event_action” - required
+* “event_name” - required
+* “event_value” - optional
+
+This example will send custom event when page element using "mybutton" ID will be clicked:
```
[the remark this answers is not in the document]
Comment (RomanSek) [reply integrations/AMP_integration.rst:51]: Moved example before params.

File: integrations/AMP_integration.rst [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+
+.. code-block:: html
```
[the remark this answers is not in the document]
Comment (RomanSek) [reply integrations/AMP_integration.rst:53]: We may prepare ful fledged HTML example using multiple triggers. GA has it in this section: https://developers.google.com/analytics/devguides/collection/amp-analytics/#complete_example

[the remark this answers is not in the document]
Comment (RomanSek) [reply integrations/AMP_integration.rst:53]: I added it at the end of document

**Review by azarecki — APPROVED** [review]

**Comment by azarecki** [comment]
This will not be a part of 15.3 release, we will merge it later without release as it is not connected to changes in release, just documentation.

**Comment by RomanSek** [comment]
It'll be merged as a https://github.com/PiwikPRO/PPMS-PublicDocs/pull/553 PR

**Comment by RomanSek** [comment]
Since release will be in 2 days we'll add it to the 15.3 docs

**Pull request merged**